docker/linux-64/Dockerfile

```diff
@@ -45,36 +45,21 @@ bison \
 python-dev \
 libxtst-dev \
 xutils-dev \
-# for synfigstudio-nsis \
```

Comment:
Maybe split it into two phases?
1) Remove support for cross-compiling with MinGW.
2) Add cross-compiling MinGW docker image.